sagemath is not yet ready to use sympy with flint ground types

With this we force ground types to gmpy2; otherwise we will get
failures when python3-flint is installed.

--- a/src/sage/env.py
+++ b/src/sage/env.py
@@ -262,6 +262,9 @@
 # Make sure mpmath uses Sage types
 os.environ['MPMATH_SAGE'] = '1'
 
+# Don't use flint in sympy
+os.environ['SYMPY_GROUND_TYPES'] = 'gmpy2'
+
 # misc
 SAGE_BANNER = var("SAGE_BANNER", "")
 SAGE_IMPORTALL = var("SAGE_IMPORTALL", "yes")
